src/verifact_agents/evidence_hunter.py

```
@@ -1,55 +1,178 @@
+            if line.strip() and not line.strip().startswith("#")
+        ]
+    except OSError as e:
+        logger.error("Error reading trust sources file %s: %s – returning empty list.", p, e)
+        return []
+    
+def get_prompt(trust_sources: list[str]):
+    """Get the prompt for the evidence hunter.
+    
+    Args:
+        trust_sources (list[str]): The list of trusted sources.
+    """
```

Ruff (0.11.9)

No blank lines allowed after function docstring (found 1)

Remove blank line(s) after function docstring

(D202)